From 2e6255f3976b67ff9f216282fe01287d7a2fc8e6 Mon Sep 17 00:00:00 2001
From: Daniel Scally <djrscally@gmail.com>
Date: Sun, 17 Jan 2021 21:39:15 +0000
Subject: [PATCH 103/103] media: i2c: Fix misnamed variable in power_down() for
 ov5693

Fix the misnamed variable in gpiod_set_value_cansleep().

Signed-off-by: Daniel Scally <djrscally@gmail.com>
---
 drivers/media/i2c/ov5693.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/ov5693.c b/drivers/media/i2c/ov5693.c
index f9ced52ad37a..9fd44a3d1d85 100644
--- a/drivers/media/i2c/ov5693.c
+++ b/drivers/media/i2c/ov5693.c
@@ -1105,7 +1105,7 @@ static int power_down(struct v4l2_subdev *sd)
 
 	dev->focus = OV5693_INVALID_CONFIG;
 
-	gpiod_set_value_cansleep(sensor->reset, 1);
+	gpiod_set_value_cansleep(dev->reset, 1);
 
 	clk_disable_unprepare(dev->clk);
 
-- 
2.17.1

